lib/exq/redis/job_queue.ex

```diff
@@ -161,17 +161,23 @@ defmodule Exq.Redis.JobQueue do
     scheduler_dequeue(redis, namespace, time_to_score(Time.now))
   end
   def scheduler_dequeue(redis, namespace, max_score) do
-    Enum.reduce(schedule_queues(namespace), 0, fn(schedule_queue, acc) ->
-      resp = Connection.zrangebyscore(redis, schedule_queue, 0, max_score)
-      case resp do
-        {:ok, jobs}      ->
-          deq_count = scheduler_dequeue_requeue(jobs, redis, namespace, schedule_queue, 0)
-          deq_count + acc
-        {:error, reason} ->
-         Logger.error("Redis error scheduler dequeue #{Kernel.inspect(reason)}}.")
-         acc
-      end
-    end)
+    queues = schedule_queues(namespace)
+    commands = Enum.map(queues, &(["ZRANGEBYSCORE", &1, 0, max_score]))
```

Comment:
Does this handle empty case when there are no queues?  If someone is using "subscribe" and "unsubscribe" that could be temporarily true.